presto-product-tests/src/main/java/io/prestosql/tests/hive/TestAvroSymlinkInputFormat.java

```diff
@@ -87,12 +74,44 @@ public void testSymlinkTable()
                 "STORED AS " +
                 "INPUTFORMAT 'org.apache.hadoop.hive.ql.io.SymlinkTextInputFormat' " +
                 "OUTPUTFORMAT 'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'");
-
+        hdfsClient.createDirectory(warehouseDirectory + "/TestAvroSymlinkInputFormat/data");
+        saveResourceOnHdfs("avro/original_data.avro", warehouseDirectory + "/TestAvroSymlinkInputFormat/data/original_data.avro");
         hdfsClient.delete(warehouseDirectory + "/test_avro_schema_symlink/symlink.txt");
```

Comment:
why `hdfsClient.delete(warehouseDirectory + "/test_avro_schema_symlink/symlink.txt");` is needed?

Reply:
It doesn't look like this directory is ever created. I will remove.